analyze.py

```diff
@@ -26,6 +21,35 @@
     "pr_comment": "/home/runner/pr_comment.txt",
 }
 
+'''
+    States on returncode
+    0 = No comment
+    1 = FAILED_COMMENT
+    5 = INCOMPLETE_COMMENT then:
+        4 = COMPLETE_SUCCESS_COMMENT
+        1 = COMPLETE_FAILED_COMMENT
+'''
+
+# Headers for distinct comment types
+DETAILS_DROPDOWN = "<details>\n<summary>Background</summary>\n<br />\nThis repository uses a GitHub Action to automatically analyze the risk of new dependencies added to requirements.txt via Pull Request. An administrator of this repository has set score requirements for Phylum's five risk domains.<br /><br />\nIf you see this comment, one or more dependencies added to the requirements.txt file in this Pull Request have failed Phylum's risk analysis.\n</details>\n\n"
```

Comment:
Is this a message specific to Python? If it's not, we might want to change `requirements.txt` to whatever the common package lock name file is for each ecosystem.

Reply:
Fixed in https://github.com/phylum-dev/phylum-analyze-pr-action/commit/f6e17e90d6087c714c5672ce77ab9a236c097c21